pkgs/top-level/all-packages.nix

```diff
@@ -24327,6 +24327,8 @@ with pkgs;
     withoutInitTools = true;
   };
 
+  tarmac = callPackage ../development/tools/tarmac { };
```

Comment:
This is the "OS-SPECIFIC" section. I don't think that was intended.

Reply:
Where should I put it?

Reply:
Is go with the section with header `### DEVELOPMENT / TOOLS`